app/ante/evm/06_account_verification.go

```diff
@@ -13,52 +13,11 @@ import (
 	evmtypes "github.com/evmos/evmos/v16/x/evm/types"
 )
 
-// EthAccountVerificationDecorator validates an account balance checks
-type EthAccountVerificationDecorator struct {
-	ak        evmtypes.AccountKeeper
-	evmKeeper EVMKeeper
-}
-
-// NewEthAccountVerificationDecorator creates a new EthAccountVerificationDecorator
-func NewEthAccountVerificationDecorator(ak evmtypes.AccountKeeper, ek EVMKeeper) EthAccountVerificationDecorator {
-	return EthAccountVerificationDecorator{
-		ak:        ak,
-		evmKeeper: ek,
-	}
-}
-
-// AnteHandle validates checks that the sender balance is greater than the total transaction cost.
+// VerifyAccountBalance checks that the sender balance is greater than the total transaction cost.
```

Comment:
nit: I suggest to be more coherent with function signature. There is no sender here:

```suggestion
// VerifyAccountBalance checks that the account balance is greater than the total transaction cost.
```